meta-xt-prod-cockpit-rcar-driver-domain/recipes-core/images/core-image-minimal.bbappend

```diff
@@ -17,6 +17,7 @@ IMAGE_INSTALL_append = " \
     vis-service \
     cluster-view \
     ${@bb.utils.contains('DISTRO_FEATURES', 'pvcamera', 'camerabe', '', d)} \
+    ${@bb.utils.contains('DISTRO_FEATURES', 'pvcamera', 'v4l2loopback-mod', '', d)} \
```

Comment:
Could this be appended to already existing condition?

${@bb.utils.contains('DISTRO_FEATURES', 'pvcamera', 'camerabe v4l2loopback-mod', '', d)} \